DEVELOPING.md

```diff
@@ -76,6 +76,8 @@ Developing the TLA⁺ Tools in Eclipse requires the [Eclipse IDE for RCP and RAP
 After you have installed Eclipse, use it open the `tlatools/org.lamport.tlatools` project directory.
 You should then be able to build, run, test, and debug the TLA⁺ Tools.
 
+Since the source files contain Unicode characters encoded in UTF-8, if using Eclipse on Windows you will need to ensure Windows Menu > Preferences > General > Workspace > "Text File Encoding" is set to UTF-8 and not Cp1252.
```

Comment:
It is probably better to enforce this on a per-project basis.  A workspace could contain other projects unrelated to TLA+.

Reply:
Is there any way to encode the setting in the .project file? I was looking at documentation [here](https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fmisc%2Fproject_description_file.html&cp=2_1_5_11) but can't find anything like that.

Reply:
Ah! I see. Right clicking project > Properties and then the Resource menu has a Text file encoding setting.

Reply:
That should create a settings file that can be added to git.